README.md

```diff
@@ -87,24 +88,24 @@ ebcli tx ethbridge create-claim --help
 
 # Now you can test out the ethbridge module by submitting a claim for an ethereum prophecy
 # Create a bridge lock claim (Ethereum prophecies are stored on the blockchain with an identifier created by concatenating the nonce and sender address)
-ebcli tx ethbridge create-claim 0 0x7B95B6EC7EbD73572298cEf32Bb54FA408207359 $(ebcli keys show testuser -a) $(ebcli keys show validator -a --bech val) 3eth lock --from=validator --chain-id=peggy --yes
+ebcli tx ethbridge create-claim 0 0x7B95B6EC7EbD73572298cEf32Bb54FA408207359 $(ebcli keys show testuser -a) $(ebcli keys show validator -a --bech val) 3stake lock --from=validator --chain-id=peggy --yes
 
 # Then read the prophecy to confirm it was created with the claim added
 ebcli query ethbridge prophecy 0 0x7B95B6EC7EbD73572298cEf32Bb54FA408207359 --trust-node
 
-# Confirm that the prophecy was successfully processed and that new eth was minted to the testuser address
+# Confirm that the prophecy was successfully processed and that new token was minted to the testuser address
 ebcli query account $(ebcli keys show testuser -a) --trust-node
 
-# Test out burning 1 of the eth for the return trip
-ebcli tx ethbridge burn $(ebcli keys show testuser -a) 0x7B95B6EC7EbD73572298cEf32Bb54FA408207359 1eth --from=testuser --chain-id=peggy --yes
+# Test out burning 1 of the token for the return trip
+ebcli tx ethbridge burn 3 0x682c2ae4053eac64cf1baaa04c739703dc043f0a $(ebcli keys show testuser -a) 0x7B95B6EC7EbD73572298cEf32Bb54FA408207359 1stake --from=testuser --chain-id=peggy --yes
```

Comment:
These cli commands are getting a bit intense, we may want to change to explicit named/flagged paramaters, like --ethereum-chain-id=3 --token-contract-address=asdf, rather than unnamed paramaters. Can do in future PR though.

Reply:
Added in https://github.com/cosmos/peggy/pull/86/commits/ea3743ad6b1a4f8031e72b636d4a2e5a6a5eab58